### ISSUE TITLE
SmartCity-Accessibility Server Side Database Branch #18 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#274]
Build 'SmartCity-Accessibility Server Side Database Branch' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Alexander Kaplan] 4b1d6d3ba05529ab9d96d48b8220253973bf0ee5 - created location builder and removed all redundant c'tors.  alot of

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Database%20Branch/18/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
